apps/src/templates/EligibilityConfirmDialog.jsx

```diff
@@ -0,0 +1,102 @@
+/** @file Confirm Dialog for Maker Discount Codes */
+import React, {Component} from 'react';
+import i18n from "@cdo/locale";
+import BaseDialog from "./BaseDialog";
+import DialogFooter from "./teacherDashboard/DialogFooter";
+import Button from "./Button";
+
+const styles = {
+  subtitle: {
+    fontSize: 16,
+  },
+  signatureBox: {
+    height: 40,
+    width: 98 + '%',
+  }
+};
+
+export default class EligibilityConfirmDialog extends Component {
+  state = {
+    signature: "",
+    activeButton: false,
+  };
+
+  verifyResponse = () => {
+    if (this.check1.checked &&
+        this.check2.checked &&
+        this.check3.checked &&
+        this.state.signature !== ""){
+      this.setState({activeButton: true});
+    } else { // if someone unchecks their responses
+      this.setState({activeButton: false});
+    }
+  }
+
+  setSignature = (event) => {
+    this.setState({signature: event.target.value});
+    this.verifyResponse();
```

Comment:
This won't work consistently - `setState` is an asynchronous operation, but `verifyResponse` depends on its result.  You can instead pass `verifyResponse` as a callback, like this:
```js
setSignature = (event) => {
  this.setState({signature: event.target.value}, this.verifyResponse);
};
```